app/uk/gov/hmrc/cataloguefrontend/connector/PrCommenterConnector.scala

```diff
@@ -38,7 +39,7 @@ object PrCommenterConnector {
     created  : Instant
```

Comment:
`Report` is used by the whole prcommenter feature (views, controllers, as well as connector) - I think adding a `prcommenter` feature package as we do for many other features makes sense - then we can move the model to that package.